src/uvw/stream.hpp

```diff
@@ -171,7 +171,9 @@ class StreamHandle: public Handle<T, U> {
     }
 
 protected:
-    using Handle<T, U>::Handle;
+	StreamHandle(std::shared_ptr<Loop> ref) noexcept
```

Comment:
`noexcept` should depend on the underlying constructor, otherwise `std::terminate` will be called in case of exceptions.